lib/personalAccessKey.ts

```diff
@@ -68,15 +71,17 @@ async function refreshAccessToken(
   );
   const { accessToken, expiresAt } = accessTokenResponse;
 
-  updateConfigAccount({
-    ...account,
-    auth: {
-      tokenInfo: {
-        accessToken,
-        expiresAt: expiresAt,
+  if (!process.env[ENVIRONMENT_VARIABLES.USE_ENVIRONMENT_HUBSPOT_CONFIG]) {
```

Comment:
Option 2: Fix this specific issue, and enforce that code should not call `updateConfigAccounts` if using env config.